generic/tests/jumbo.py

```diff
@@ -176,11 +176,18 @@ def size_increase_ping(step=random.randrange(90, 110)):
                                                  packetsize=size,
                                                  hint="do", timeout=1)
                 if status != 0:
-                    status, output = utils_test.ping(guest_ip, 10,
-                                                     packetsize=size,
-                                                     adaptive=True,
-                                                     hint="do",
-                                                     timeout=20)
+                    if params.get("min_mtu") == "yes":
+                        status, output = utils_test.ping(guest_ip, 10,
+                                                         packetsize=size,
+                                                         adaptive=True,
+                                                         hint="want",
```

Comment:
It is possible to set only different `hint` values, because other parameters are the same.

Reply:
Changed it, pelase help me review it again. Thanks a lot.